Airline_Web_API/Airline_Web_API/Services/AirportService.cs

```diff
@@ -40,5 +49,57 @@ public async Task<IEnumerable<AirportViewModel>> GetAirports(string value)
 
             return results;
         }
+
+        public async Task AddAirportAsync(Airport airport)
+        {
+            _context.Airports.Add(airport);
+            await _context.SaveChangesAsync();
+        }
+
+        public async Task<Response<string>> UpdateAirportAsync(AirportViewModel model)
+        {
+            var airport = await _context.Airports.FindAsync(model.Id);
+
+            if (airport == null)
+            {
+                return new Response<string>
+                {
+                    Success = false,
+                    Message = "There is no such airport"
+                };
+            }
+
+            _mapper.Map(model, airport);
+            await _context.SaveChangesAsync();
+
+            return new Response<string>
+            {
+                Success = true,
+                Message = "Airport was succesfully updated"
+            };
+        }
+
+        public async Task<Response<string>> DeleteAirportAsync(AirportViewModel model)
+        {
+            var airport = await _context.Airports.FindAsync(model.Id);
+
+            if (airport == null)
+            {
+                return new Response<string>
+                {
+                    Success = false,
+                    Message = "There is no such airport"
+                };
+            }
+
+            _context.Remove(airport);
```

Comment:
What are you going to do with all the entities that depend on airport, e.g. flights, tickets, etc? Are you going to delete them as well? What are you going to show in user tickets history if it contains some deleted airports?

Reply:
It's not finished version. I'll add logical delete using field 'status'

Reply:
And I would prefer to add it in new PR as I'll add this property not only for airport

Reply:
Ok, please create separate task for this.